README.md

```diff
@@ -54,74 +77,158 @@ Outputs:
+* `useGRMtoFitNULL`: same as above, but without the sparse option (default: `FALSE`).
+* `isCovariateOffset`: use covariates as offset allows to only estimate coefficients once and reduce cost (check that the model retains calibration, in which case set to `TRUE` for cost optimisation).
+* `isCovariateTransform`: transform (explain) covariates (default: `TRUE`).
+* `skipModelFitting`: boolean (default: `FALSE`).
+* `tol`: convergence tolerance (default: 0.00001, which works well in our hands, we could test with a larger number to decrease cost).
+* `maxiterPCG`: convergence max number of iterations (default: 500 but increase to 5,000 for the specific genes whose job does not converge).
+* `IsOverwriteVarianceRatioFile`: if the file already exists, skip or overwrite, default is the latter (i.e. default: `TRUE`).
 
 Single-variant association testing ([common variants step 2](https://weizhou0.github.io/SAIGE-QTL-doc/docs/single_step2.html)):
 
-* ```vcf_file```: path to VCF file containing genetic variants to be tested
-* ```vcf_file_index```: corresponding .csi index file (not .tbi)
-* ```vcf_field```: DS for dosages, GT for genotypes (default = 'GT')
-* ```saige_output_file```: path to output file
-* ```chrom```: chromosome to be tested
-* ```cis_window_file```: path to file specifying cis window / region to test (generated in part 2 of the pipeline, get anndata script)
-* ```gmmat_model_path```: path to estimated null model (.rda) generated in step 2
-* ```variance_ratio_path```: path to variance ratio txt file generated in step 1
-* ```min_maf```: minimum minor allele frequency (MAF) (default: 0)
-* ```min_mac```: minimum minor allele count (MAC) (default: 5)
-* ```loco_bool```: boolean specifying whether leave-one-chromosome-out should be used (default: ```FALSE```)
-* ```n_markers```: internal parameter to batchify variants tested (default: 10000),
-* ```spa_cutoff```: internal parameter to do with the saddlepoint approximation, does not make much of a difference for us (default: 10000),
+In script (`saige_assoc.py`):
+
+* `vcf_file`: path to VCF file containing genetic variants to test.
+* `vcf_file_index`: corresponding .csi index file (not .tbi). check, just prefix?
+* `saige_output_file`: path to output file (check name).
+* `chrom`: chromosome to test.
+* `cis_window_file`: path to file specifying cis window / region to test (generated in part 2 of the pipeline, get anndata script).
+* `gmmat_model_path`: path to estimated null model (.rda) generated in step 1.
+* `variance_ratio_path`: path to variance ratio txt file generated in step 1.
+
+In config (under `[saige.sv_test]` in `saige_assoc_test.toml`):
+
+* `vcfField`: DS for dosages, GT for genotypes (default: `GT`).
+* `minMAF`: minimum minor allele frequency (MAF) (default: 0).
+* `minMAC`: minimum minor allele count (MAC) (default: 5).
```

Comment:
What happens when both values are set? Does it take the largest?

Reply:
good question, I assume so? I'll check.

Reply:
all done